integration_tests/src/examples.rs

```diff
@@ -118,11 +116,23 @@ pub fn download_file(url: &str, path: &str) -> Result<PathBuf> {
     Ok(sample_path)
 }
 
-#[allow(dead_code)]
-pub fn start_ffplay(ip: &str, video_port: u16, audio_port: Option<u16>) -> Result<()> {
-    let output_sdp_path = match audio_port {
-        Some(audio_port) => write_video_audio_example_sdp_file(ip, video_port, audio_port),
-        None => write_video_example_sdp_file(ip, video_port),
+// plays H264 encoded video and Opus encoded audio using ffmpeg. If no port given, returns Error
```

Comment:
1. Use docs (`///`), not comments. Generally:
- docs - explain how to use code
- comments - describe why code is written in a certain way / what is happening (most of the time you should do it with var names/functions etc., but sometimes it's easier to leave a comment). This should be read by people reading/modifying your code, not just using/calling it.
2. This type of code should be self-explanatory/self-documenting. Use function/variable names that describe purpose instead of using docs. In this case, `start_ffplay` is good enough for me. 
```suggestion
```